attendance.py

```diff
@@ -9,4 +13,30 @@ def main():
     tk.Label(master=window, text='ahmed is not changing the stuff',
              font='Arial 20 roman normal').pack()
 
+    def getCSV(filename):                                               #function for getting encodings from csv files
+        with open(f"./{directoryName}/{filename}") as f:
+            reader = csv.reader(f)
+            encodings = []
+            for code in reader:
+                encodings.extend(code)
+            encodings = [float(code) for code in encodings]
+            return encodings
+
+    knownEncodingFiles = os.listdir(f"./{directoryName}")               #Getting names of all csv files in directory
+    encodings=[]                                                        #variable definition
+    encodingPrimaryKey=[]                                               #variable definition
+    for file in knownEncodingFiles:                                     #Looping over the files to extract encodings
+        encodings.append(getCSV(file))                                  #Saving encodings in encodins list
```

Comment:
instead of this, we use this syntax which does both of these in one step, encodings[file[:-3]] = getCSV(file)

Reply:
My code is easier on the eyes but whatever you say big boss.

Reply:
Yeah it is really good but we should use the abstraction that library provides us...